components/form/Toggle.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { useFormContext } from 'react-hook-form';
+import PropTypes from 'prop-types';
+import { Label, ErrorMessage } from '.';
+import classNames from 'classnames';
+
+const Toggle = ({ containerClassName, label, name, value, toggleClassName }) => {
+  const { register } = useFormContext();
+  return (
+    <>
+      <div className={classNames('relative flex items-center', containerClassName)}>
+        <input
+          name={name}
+          type="checkbox"
+          {...register(name)}
+          id={`toggle-${name}-${value.toString()}`}
```

Comment:
We can use the name directly here as toggle is not meant to be used as a group like the checkbox. Toggle is a standalone component like the Input component.

```suggestion
          id={name}
```